lib/UserAgentApplication.ts

```diff
@@ -1098,7 +1098,7 @@ namespace Msal {
                                     "The user object received in the response is the same as the one passed in the acquireToken request");
                             } else {
                                 this._requestContext.logger.warning(
-                                    "The user object received in the response is not same as the one passed in the acquireToken request");
+                                    "The user object created from the response is not the same as the one passed in the acquireToken request");
```

Comment:
>"The user object created from the response is not the same as the one passed in the acquireToken request") [](start = 36, length = 106)

nit: This is not 100% true. User could have been null or acquireTokenUser could have been null